File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.2/JMBA_Olo_Android_SDK/app/src/main/java/com/wearehathway/apps/olo/Models/OloUser.java
package com.wearehathway.apps.olo.Models;

/**
 * Created by Nauman Afzaal on 27/04/15.
 */
public class OloUser
{
    private String firstname;
    private String lastname;
    private String emailaddress;
    private String authtoken;
    private String contactnumber;
    private String reference;
    private String basketid;
    private String cardsuffix;

    public OloUser(String firstName, String lastName, String emailaddress, String cardsuffix)
    {
        this.firstname = firstName;
        this.lastname = lastName;
        this.emailaddress = emailaddress;
        this.cardsuffix = cardsuffix;
    }

    public String getFirstname()
    {
        return firstname;
    }

    public String getLastname()
    {
        return lastname;
    }

    public String getEmailaddress()
    {
        return emailaddress;
    }

    public String getAuthtoken()
    {
        return authtoken;
    }

    public String getContactnumber()
    {
        return contactnumber;
    }

    public String getReference()
    {
        return reference;
    }

    public String getBasketid()
    {
        return basketid;
    }

    public String getCardsuffix()
    {
        return cardsuffix;
    }
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.1/JMBA_Olo_iOS_SDK/Olo/OloUser.swift
//
//  OloUser.swift
//  JambaJuice
//
//  Created by Eneko Alonso on 4/22/15.
//  Copyright (c) 2015 Jamba Juice. All rights reserved.
//

//  OLO User Response
//
//    {
//        firstname (string): ,
//        lastname (string): ,
//        emailaddress (string): ,
//        authtoken (string): ,
//        contactnumber (string): ,
//        reference (string, optional): can be used to associate an external Id,
//        basketid (string, optional): if specified will link the specified basket to this user
//    }

import UIKit
import SwiftyJSON

public struct OloUser {
   
    public var authToken: String?
    public var firstName: String
    public var lastName: String
    public var emailAddress: String
    public var contactNumber: String?
    public var reference: String?
    public var baskedId: String?
    public var cardSuffix: String?

    public init(json: JSON) {
        authToken      = json["authtoken"].string
        firstName      = json["firstname"].stringValue
        lastName       = json["lastname"].stringValue
        emailAddress   = json["emailaddress"].stringValue
        contactNumber  = json["contactnumber"].string
        reference      = json["reference"].string
        baskedId       = json["basketid"].string
        cardSuffix     = json["cardsuffix"].string
    }
    
    public init(firstName: String, lastName: String, emailAddress: String) {
        self.firstName = firstName
        self.lastName = lastName
        self.emailAddress = emailAddress
    }
    
}
